src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -144,5 +146,33 @@ export const releaseHead = (url, data) => (dispatch) => {
       .catch(error => {
         notify.show(`${error.response.data.__all__[0]}`, 'error', 6000);
       })
-    }
+};
+
+export const createVersion = (url, data) => (dispatch) => {
+  return api.dictionaries
+    .creatingVersion(url, data)
+    .then(
+      (payload) => {
+        dispatch(creatingVersionsSuccess(payload));
+        dispatch(creatingVersionsError());
+        notify.show(
+          `${payload.id} has successfully been released`,
+          'success', 6000,
+        );
+      }
+    )
+    .catch((error) => {
+      if(error.response.data.detail){
+      dispatch(creatingVersionsError(error.response.data));
```

Comment:
Good job. However, it seems the indentation isn't properly done on the `catch` method.

Reply:
Thank you the observation. It has now been fixed.

Reply:
I think the `if` and `else` blocks from  [line 164](https://github.com/openmrs/openmrs-ocl-client/blob/d3a9529be7d9f6f84d88acbe2f2b0819ffeb0deb/src/redux/actions/dictionaries/dictionaryActionCreators.js#L164) to [line 176](https://github.com/openmrs/openmrs-ocl-client/blob/d3a9529be7d9f6f84d88acbe2f2b0819ffeb0deb/src/redux/actions/dictionaries/dictionaryActionCreators.js#L176)  needs to be properly indented.

Reply:
Ooh. Now I see it! Thank you!